site/app/templates/submission/regrade/Discussion.twig

```diff
@@ -69,7 +69,7 @@
 {% for post in posts %}
     <div style="margin-top: 20px;">
         <div class='{{ post.is_staff ? "post_box important" : "post_box" }} {{ loop.index0 == 0 ? "first_post" : "" }}' style="padding:20px;">
-            <p style='white-space:pre-wrap;'>{{ post.content }}</p>
+            <p style='white-space:pre-wrap;'>{% include "forum/RenderPost.twig" with { "post_content": post.content } only %}</p>
```

Comment:
It looks like that `RenderPost.twig` is added in #4000 (which is not merged yet). If that's true I think it would be better to explicitly point it out (like marking this as WIP or leaving a comment here) in case that it is approved and accidentally merged into master branch w/o `RenderPost.twig`.

Reply:
Mentioned! Thanks!